hack/update-manifests.sh

```diff
@@ -35,17 +37,22 @@ cd ${SRCROOT}/manifests/base && $KUSTOMIZE edit set image quay.io/argoproj/argoc
 cd ${SRCROOT}/manifests/ha/base && $KUSTOMIZE edit set image quay.io/argoproj/argocd=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG}
 cd ${SRCROOT}/manifests/core-install && $KUSTOMIZE edit set image quay.io/argoproj/argocd=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG}
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/install.yaml"
+echo -n "" > "${SRCROOT}/manifests/install.yaml"
 $KUSTOMIZE build "${SRCROOT}/manifests/cluster-install" >> "${SRCROOT}/manifests/install.yaml"
+go run $HACK_IMAGEPULLPOLICY -manifest=${SRCROOT}/manifests/install.yaml -image=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG} -image-pull-policy=$IMAGE_PULL_POLICY
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/namespace-install.yaml"
+echo -n "" > "${SRCROOT}/manifests/namespace-install.yaml"
 $KUSTOMIZE build "${SRCROOT}/manifests/namespace-install" >> "${SRCROOT}/manifests/namespace-install.yaml"
+go run $HACK_IMAGEPULLPOLICY -manifest=${SRCROOT}/manifests/namespace-install.yaml -image=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG} -image-pull-policy=$IMAGE_PULL_POLICY
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/ha/install.yaml"
+echo -n "" > "${SRCROOT}/manifests/ha/install.yaml"
 $KUSTOMIZE build "${SRCROOT}/manifests/ha/cluster-install" >> "${SRCROOT}/manifests/ha/install.yaml"
+go run $HACK_IMAGEPULLPOLICY -manifest=${SRCROOT}/manifests/ha/install.yaml -image=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG} -image-pull-policy=$IMAGE_PULL_POLICY
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/ha/namespace-install.yaml"
+echo -n "" > "${SRCROOT}/manifests/ha/namespace-install.yaml"
 $KUSTOMIZE build "${SRCROOT}/manifests/ha/namespace-install" >> "${SRCROOT}/manifests/ha/namespace-install.yaml"
+go run $HACK_IMAGEPULLPOLICY -manifest=${SRCROOT}/manifests/ha/namespace-install.yaml -image=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG} -image-pull-policy=$IMAGE_PULL_POLICY
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/core-install.yaml"
+echo -n "" > "${SRCROOT}/manifests/core-install.yaml"
```

Comment:
Why do we change these?

Reply:
Hello, @andrii-korotkov-verkada.

I created `hack/update-manifests-imagepullpolicy/main.go` to update the `imagePullPolicy` value for the `quay.io/argoproj/argocd` image.
This script parses the manifests YAML file (built by kustomize), updates the `imagePullPolicy` value if needed, and then regenerates/overwrites the manifests YAML file.
When regenerating, it writes the `AUTOGENMSG`, so I removed that in `hack/update-manifests.sh`.

If you think the message should still be included, I’m happy to revert to the original version. :)
Let me know what you think!

Thanks for your feedback!